src/Python/Filtering/TriangulateTerrainMap.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python
+
+import vtk
+import random
+
+def main():
+
+    colors = vtk.vtkNamedColors()
+    # Create points on an XY grid with random Z coordinate
+    points = vtk.vtkPoints()
+    
+    for x in range (0, 10):
+        for y in range (0, 10): 
+           points.InsertNextPoint(x, y, random.uniform(0.0, 3.0))
+      
```

Comment:
I suggest removing random and using `vtkMinimalStandardRandomSequence` instead. Look at [DelaunayMesh.py](https://lorensen.github.io/VTKExamples/site/Python/Modelling/DelaunayMesh/) for how to use it.

Reply:
@ajpmaclean Thank You! I have made the changes as suggested. Please review on the same.

Reply:
@ajpmaclean Sure! I will do it

Reply:
@ajpmaclean I have updated. Please review on the same.